README.md

```diff
@@ -131,11 +131,11 @@ If your PR is passing all checks, but has not been acted on by the staged recipe
 maintainers, you can ping @conda-forge/staged-recipes to request action. You do
 not need to wait any specific amount of time once the recipe is ready to go.
 
-If your recipe still does not recieve any attention after a few days, you may
+If your recipe still does not receive any attention after a few days, you may
```

Comment:
Please do not modify files in a recipe PR outside of the recipes dir.  Please make a separate PR for this file, if you want. Thanks!

Reply:
Thank you for the hint. I reverted the spelling commit. Now the recipe should be fine.